src/Sulu/Bundle/SecurityBundle/Security/Exception/EmailNotUniqueException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Security\Exception;
+
+/**
+ * This Exception is thrown if the email for a user is not unique
+ * @package Sulu\Bundle\SecurityBundle\Security\Exception
+ */
+class EmailNotUniqueException extends SecurityException
+{
+    public function __construct()
+    {
+        parent::__construct('security.user.error.notUniqueEmail', 1004);
```

Comment:
This message should not be translated... It should contain an english message with the non-unique email for the developer, it should not be used for the error messages in the frontend.